findbugs.properties

```diff
@@ -45,7 +45,12 @@ defaults.mavenArtifactId = sonar-findbugs-plugin
 4.2.1.changelogUrl = https://github.com/spotbugs/sonar-findbugs/releases/tag/4.2.1
 4.2.1.downloadUrl = https://repo.maven.apache.org/maven2/com/github/spotbugs/sonar-findbugs-plugin/4.2.1/sonar-findbugs-plugin-4.2.1.jar
 4.2.2.description = Use SpotBugs 4.7.3, sb-contrib 7.4.7, and findsecbugs 1.12.0
-4.2.2.sqVersions = [8.9,LATEST]
+4.2.2.sqVersions = [8.9,9.9.*]
 4.2.2.date = 2022-10-20
 4.2.2.changelogUrl = https://github.com/spotbugs/sonar-findbugs/releases/tag/4.2.2
 4.2.2.downloadUrl = https://repo.maven.apache.org/maven2/com/github/spotbugs/sonar-findbugs-plugin/4.2.2/sonar-findbugs-plugin-4.2.2.jar
+4.2.3.description = Use SpotBugs 4.7.3, sb-contrib 7.4.7, and findsecbugs 1.12.0
+4.2.3.sqVersions = [8.9,LATEST]
+4.2.3.date = 2023-02-12
+4.2.3.changelogUrl = https://github.com/spotbugs/sonar-findbugs/releases/tag/4.2.3
```

Comment:
Normally I don't know enough to comment on the contents of a changelog, but...
I would have expected to see [#18](This should be fixed in version 4.2.3 (just released)
Note that the fix needs the SonarQube version to be >= 9.8) listed in the changelog and I don't...?

Reply:
I'm not sure if it's a valid justification but historically the "description" was always listing the SpotBugs (& plugins) versions, without any other details.
If the "description" is actually meant to be a changelog, I suppose I can add more info here. Let me know what you think!

Reply:
I'm for whatever gives users access to a comprehensive list of changes. I only question this because what I end up at (https://github.com/spotbugs/sonar-findbugs/compare/4.2.2...4.2.3) doesn't include something that I was pointed to this version for...?

Reply:
The changelog referenced here ends with
> Full Changelog: https://github.com/spotbugs/sonar-findbugs/compare/4.2.2...4.2.3